spec/models/metric_spec.rb

```diff
@@ -0,0 +1,50 @@
+RSpec.describe Metric do
+  let(:metric_data) do
```

Comment:
Would you mind moving the `let` closer to the scope describe where it is used? Would it be also possible to move it below the `it` so we prevent scrolling when reading the test?

Reply:
Done https://github.com/alphagov/content-data-admin/pull/328/commits/c3e2177aa0f01deae437b8b82a96ac961cfd4fc5